docker/development-easy-arm/docker-compose.yml

```diff
@@ -68,14 +80,32 @@ services:
       OPENEMR_SETTING_gbl_ldap_host: 'ldap://openldap:389'
       OPENEMR_SETTING_gbl_ldap_dn: 'cn={login},dc=example,dc=org'
     depends_on:
-    - mysql
+      mysql:
+        condition: service_healthy
+    healthcheck:
+      test:
+      - CMD
+      - /usr/bin/curl
+      - --silent
+      - --fail
+      - --show-error
+      - --location
+      - http://localhost/
```

Comment:
Wouldn't it be better to hit https here? instead of http?  My preference is to make sure the SSL is working versus regular http.